cl/recap/views.py

```diff
@@ -56,7 +56,10 @@ class PacerProcessingQueueViewSet(LoggingMixin, ModelViewSet):
     async def perform_create(self, serializer):
         pq = await sync_to_async(serializer.save)(uploader=self.request.user)
         recap_upload_task = asyncio.create_task(process_recap_upload(pq))
-        await asyncio.shield(recap_upload_task)
+        try:
+            await asyncio.shield(recap_upload_task)
+        except asyncio.CancelledError:
+            await recap_upload_task
```

Comment:
Could you put a comment on this, since it's going to baffle people in the future? Maybe a link to something?

Reply:
comment added